### PR TITLE
Add DeepEdit Bundle

### PR DESCRIPTION
**Pull request by diazandr3s**
Signed-off-by: Andres Diaz-Pinto <diazandr3s@gmail.com>

DeepEdit Bundle

### CODE REVIEW

**Comment by diazandr3s** [comment]
@Nic-Ma how I can send a dictionary to a transform?

When using this transform [here](https://github.com/diazandr3s/tutorials/blob/deepeditBundle/deepedit/deepedit_bundle/configs/train.json#L51)

https://github.com/Project-MONAI/MONAI/blob/dev/monai/apps/deepedit/transforms.py#L101

I'm getting this error:

>     for idx, (key_label, val_label) in enumerate(self.label_names.items(), start=1):
> AttributeError: 'str' object has no attribute 'items'

File: deepedit/deepedit_bundle/configs/train.json [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+                "%train#pre_transforms#transforms#0",
+                "%train#pre_transforms#transforms#1",
+                "%train#pre_transforms#transforms#2",
+                "%train#pre_transforms#transforms#3",
+                "%train#pre_transforms#transforms#4",
+                "%train#pre_transforms#transforms#10",
+                "%train#pre_transforms#transforms#11",
+                "%train#pre_transforms#transforms#12",
+                "%train#pre_transforms#transforms#13",
+                "%train#pre_transforms#transforms#14"
```
Comment (wyli) [inline deepedit/deepedit_bundle/configs/train.json:282]: any idea how to simplify this? perhaps put 0-4 and 10-14 to separate .json files? cc@Nic-Ma the current one is not easy to read...

Comment (ericspod) [reply deepedit/deepedit_bundle/configs/train.json:282]: One way to do this is have a separate definition called `common_transforms` which is just a list of transforms common to both training and validation, then textually expand that list here using the % sigil and in `@train#pre_transforms`.

Comment (Nic-Ma) [reply deepedit/deepedit_bundle/configs/train.json:282]: I will update the spleen bundle example for this part.

Thanks.

Comment (Nic-Ma) [reply deepedit/deepedit_bundle/configs/train.json:282]: Hi @ericspod ,

Submitted a PR for it: https://github.com/Project-MONAI/tutorials/pull/710.
Could you please help review it?

Thanks in advance.

Comment (Nic-Ma) [reply deepedit/deepedit_bundle/configs/train.json:282]: Hi @diazandr3s ,

Could you please help update this bundle example referring to the updated spleen bundle?

Thanks in advance.

Comment (Nic-Ma) [reply deepedit/deepedit_bundle/configs/train.json:282]: Hi @diazandr3s ,

For your case, I think you can chain the transforms for the trainer like:
```json
"preprocessing": {
    "_target_": "Compose",
    "transforms": "$@train#deterministic_trans[0: 5] + @train#random_trans + deterministic_trans[5: 10]"
}
```
And BTW, I already added you to the `model-zoo` repo, could you please help submit this bundle to that bundle?
https://github.com/Project-MONAI/model-zoo
Then let's close this PR and discuss in `model-zoo` directly.

Thanks in advance.

Comment (diazandr3s) [reply deepedit/deepedit_bundle/configs/train.json:282]: Thanks, @Nic-Ma
Will modify this and create the PR there :)

**Comment by diazandr3s** [comment]
Closing this PR. 
A new one has been created here: https://github.com/Project-MONAI/model-zoo/pull/3

**Pull request closed without merging**